raw-data/SAPHON/saphon20121031.tsv

```diff
@@ -126,7 +126,7 @@ o	o	o	o	o	o	o	o	g	g	g	g	g	g	g	g	g	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f	f
 1926	Cayubaba	Cayubaba		Cayubaba	cyb	Bolivia	Isolate	2108	-13.4205	-65.4641							0	0	1	0	0	0	0	0	0	0	0	1		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0		0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	1	0		0	0	0		0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0		0	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1	1		0	0	1	0	0	0	0		0	0	0	0	0	0	0	0	0		1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	0	0	0	0	0	0	0		0	1	0	0	1	0	0	0	0	0	0	0	1	1	0	0	1	0	0	1	0	0	0	0	0	0				0	0	0	0	1	1	0	0	1	0	0	1	0	0	0	0	0	0		0	1	0	0	1	0	0	0	1	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0		0	0	0		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	0	1	1	0	0	0	0	0	0	0	0	0	0	0	Aikhenvald, Alexandra  and R.M.W. Dixon. 1999. Other small families and isolates. In R.M.W. Dixon and Alexandra Aikhenvald (eds.), The Amazonian Languages, pp. 341-384. Cambridge University Press.												
-1927	Mỹky	Mỹky	Irántxe	Irantxe	irn	Brazil	Isolate	417	-13	-58							0	0	1	0	0	0	1	0	0	0	0	1		0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0		1	0	0		0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	0	0	0	1	0	0		0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1	0		0	0	0	0	0	0	0		0	0	0	1	0	0	1	0	0		1	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	1	1	0	0	1	0	0	0	0		0	1	0	0	1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	1	1	0	1	1	0	0				0	0	0	0	0	0	0	0	1	0	0	1	1	0	0	1	1	0		0	1	0	0	1	1	0	0	1	0	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0		0	0	0		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0			0	0	0	0	1	1	0	0	1	0	1	0	0	0	0	0	0	0	0	0	0	0	Meader, R.E. 1967. Iranxe: notas gramaticais e lista vocabular. Rio de JaneiroL Museu Nacional and Universidade Federal do Rio de Janeiro.												
+1927	Mỹky	Mỹky	Irántxe	Irantxe	irn	Brazil	Isolate	417	-13	-58							0	0	1	0	0	0	1	0	0	0	0	1	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	0	0	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1	0		0	0	0	0	0	0	0	0	0	0	0	1	0	0	1	0	0	0	1	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	1	1	0	0	1	0	0	0	0	0	0	1	0	0	1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	1	1	0	0	0	0	0	0	0	1	0	0	1	1	0	0	1	0	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	0	0	1	0	1	0	0	0	0	0	0	0	0	0	0	0	Meader, R.E. 1967. Iranxe: notas gramaticais e lista vocabular. Rio de JaneiroL Museu Nacional and Universidade Federal do Rio de Janeiro.												
```

Comment:
Yes. I just noticed that there were a few cells without 1 or 0 and it struck me as inconsistent -- and since we're changing the contents of this upstream data source, I decided to add them to be consistent. (At some point it would make sense to get and integrate the latest version of SAPHON, which I'm assuming has fixes and more languages.)